ls1intum/Artemis
# Simplify Java programming exercise templates based on new artemis-java-test-sandbox version

### Checklist

### Motivation and Context
Simplify the template, move test classes (which do not change between projects) into artemis-java-test-sandbox and use them based on the downloaded maven dependency

### Steps for Testing
1. Create multiple programming exercises. Create in total 6 exercises:
- Java/ Non-Sequential/ Maven project type
- Java/ Non-Sequential/ Eclipse project type
- Java/ Sequential/ Maven project type
- Java/ Sequential/ Eclipse project type
- Kotlin/ Non-Sequential
- Kotlin/ Sequential
2. Try out that the tests for template and solution are still working properly
3. Participate in the exercises
4. Make a submission and check that everything still works

Related artemis-java-test-sandbox PR: https://github.com/ls1intum/artemis-java-test-sandbox/pull/62
